### DISCUSSION TITLE
v4

### DISCUSSION THREAD

**Discussion by fabiocbr75** [VIP-Logic/NBomber#518] [General]
Hello,
I really appreciate NBomber.
What are the news of the version 4?
Are there a lot of breaking changes in comparison to version 3?

Thanks

**Comment by AntyaDev** [comment]
Hello,
I am working on documentation for v4. I hope I will finish the part about the basic API this week.
It has some breaking changes, for sure.
I suggest checking the [examples folder](https://github.com/PragmaticFlow/NBomber/blob/dev/examples/CSharpProd/Program.cs).
